libr/core/cmd_write.c

```diff
@@ -534,6 +520,75 @@ static void cmd_write_value (RCore *core, const char *input) {
 	r_core_block_read (core);
 }
 
+static RCmdStatus common_wv_handler(RCore *core, int argc, const char **argv, int type) {
+	ut64 off = 0LL;
+	ut8 buf[sizeof(ut64)];
+	int wseek = r_config_get_i (core->config, "cfg.wseek");
+	bool be = r_config_get_i (core->config, "cfg.bigendian");
+
+	core->num->value = 0;
+	if (argc != 2) {
+		return R_CMD_STATUS_INVALID;
```

Comment:
shall we already think about introducing a new `R_CMD_STATUS_WRONG_ARGS`? I think it makes sense, as you don't want a general error in this case, but probably you want the RCore to print the help of this command or a more meaningful message anyway. @karliss wdyt?

Reply:
Seems somewhat reasonable. Will need to update the error update macro.